src/main/java/com/spotify/reaper/cassandra/JmxProxy.java

```diff
@@ -252,9 +253,12 @@ public BigInteger apply(String s) {
   @NotNull
   public Map<String, String> getEndpointToHostId() {
     checkNotNull(ssProxy, "Looks like the proxy is not connected");
-    Map<String, String> hosts =
-        ((StorageServiceMBean) ssProxy).getEndpointToHostId();
-
+    Map<String, String> hosts = Maps.newHashMap();
+    try{
+      hosts = ((StorageServiceMBean) ssProxy).getEndpointToHostId();
+    } catch(UndeclaredThrowableException e){
+      hosts = ((StorageServiceMBean) ssProxy).getHostIdMap();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "getHostIdMap"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)